Build optimization

Kiuh opened the pull request:
- Correct texture compression
- Correct audio compression
- Remove unused assets
- Remove unused BuildIn packages

Kiuh committed:
Introduce build size optimization

- Correct texture compression
- Correct audio compression
- Remove unused assets

Kiuh committed:
Remove unused buildin packages and remove unused fonts

Kiuh committed:
Remove more unused assets and fix some textures

Adammui reviewed (changes requested):
Visual bug with mouse body visible while appearing:

https://github.com/user-attachments/assets/8d13fbf4-a57d-4128-a348-df8fe9f1e35f

Kiuh committed:
Fixes by review + small asset rearranging

Adammui reviewed (changes requested):
Overlap of masks
![image](https://github.com/user-attachments/assets/627fa005-afe7-4037-a005-c4cd11dc63b0)

![image](https://github.com/user-attachments/assets/359f8389-82b5-4491-9eac-d1559893d02d)

Kiuh committed:
Review and bug fixes

- Update Odin version
- Fix barn transition bug
- Fix audio sliders mask in UI bug
